gmail/Package.md

```diff
@@ -1,79 +1,112 @@
-# Gmail Connector
-
+# Package Overview
 Connects to Gmail from Ballerina. 
 
-The Gmail connector provides an optimized way to use the Gmail REST API from your Ballerina programs. 
-It handles OAuth 2.0 and provides auto completion and type conversions.
-
-## Compatibility
-
-| Ballerina Language Version                   | Gmail API Version |  
-| :-------------------------------------------:|:-----------------:| 
-| 0.970.0-beta13                               | v1                | 
-
-## Getting started
-
-1.  To download and install Ballerina, see the [Getting Started](https://ballerina.io/learn/getting-started/) guide.
+This package provides a Ballerina API for the Gmail REST API. It provides the ability to send emails, read emails, 
+delete emails, read threads, get the Gmail profile, etc. It handles OAuth 2.0 and provides auto completion and 
+type conversions.
 
-2.  Obtain your OAuth 2.0 credentials. To access a Gmail endpoint, you will need to provide the Client ID, 
-    Client Secret, and Refresh Token, or just the Access Token. For more information, see the [Gmail OAuth 2.0 
-    documentation](https://developers.google.com/identity/protocols/OAuth2).
+**Message Operations**
 
-3. Create a new Ballerina project by executing the following command.
+The wso2/gmail package contains operations to send emails in Text and HTML formats with attachments and inline images. 
+It supports searching and reading messages in Gmail using Gmail filters. The package also supports trashing, untrashing, 
+and deleting messages as well.
```

Comment:
Make sure this sentence isn't splitting onto another line when it's displayed

Reply:
It is not split to a new line when displaying.